frappe/integrations/utils.py

```diff
@@ -29,7 +29,10 @@ def make_request(method: str, url: str, auth=None, headers=None, data=None, json
 		else:
 			return
 	except Exception as exc:
-		frappe.log_error()
+		try:
```

Comment:
```suggestion
		if frappe.flags.integration_request_doc:
```
Seems simpler to just check if its set I guess? 
Also I think the current logic would log the `AttributeError` and not the request error, since log_error() will rely on `sys.exc_info()` for getting exception details. So maybe just use 2 flags to set the doctype and docname here, and pass to log_error if present (that's what `doc.log_error()` does basically)

Reply:
Absolutely! Thanks!

Force-pushed [here](https://github.com/frappe/frappe/compare/119d61f85982eb930342cd6203112b35a21cdf4d..f9cdcfb7af33d6aea99c166fddbbc051492bd5be)